include/nuttx/tls.h

```diff
@@ -66,6 +66,8 @@
 #  define TLS_INFO(sp)     ((FAR struct tls_info_s *)((sp) & ~TLS_STACK_MASK))
 #endif
 
+#define TLS_THREAD_EIXT     (1 << 0)
```

Comment:
Typo: EIXT

Suggest naming this to TLS_THREAD_EXIT_PROCESSING as per https://github.com/apache/nuttx/blob/38ddf7d59f9e1a7aa98b17e0ee4793b41ff18ef1/include/nuttx/sched.h#L127

Reply:
done